docs/core-development/ide-tools.md

```diff
@@ -0,0 +1,120 @@
+IDE Tools for developing Bolt
+=============================
+
+Currently the core team members uses a combination of PhpStorm/IntelliJ, vim,
+Atom, Sublime, Netbeans, and Visual Studio Code. There is probably even an 
+emacs die-hard out there.
+
+In no way should any of these this be seen as a requirement to develop on
+Bolt's core code, but some tools will make your life easier.
+
+This document currently covers the following IDEs:
+  * PhpStorm & IntelliJ IDEA
+
+## PhpStorm & IntelliJ IDEA
+
+### Useful Plugins
+
+* Silex Plugin
+* Symfony Plugin
+* Twig Support
+
+In particular, most developers will find the Silex plugin extremely useful, as
+it will allow services stored in the application container.
```

Comment:
I assume something like "to be used with autocompletion and code inspection" should be added, right?